condor.go

```diff
@@ -269,6 +273,10 @@ func (cl *CondorLauncher) handleLaunchRequests(condorPath, condorConfig string)
 				if err != nil {
 					log.Errorf("%+v\n", errors.Wrap(err, "failed to publish launch failure job update"))
```

Comment:
We probably don't want this to send a `Failed` state until the second try. Our system doesn't let you go into "running" after having already gone into "failed", so it'd mean if the first try failed, the job would actually run but never go into running or completed in the UI.

Of course, as part of the async analyses stuff we talked about adding more states, which would make this more normal -- a failure at this stage would just be an extra update in the "processing" state (or whatever we decided to call it) before going into the "queued" state (or whatever we decided to call it).

Reply:
Good catch 👍